src/Button/Button.react.js

```diff
@@ -89,18 +91,19 @@ function getStyles(props, context, state) {
         container: [
             button.container,
             !raised && buttonFlat.container,
-            disabled && buttonDisabled.container,
             raised && buttonRaised.container,
             local.container,
             props.style.container,
+            (disabled && raised) && buttonRaisedDisabled.container,
```

Comment:
Running linter now... as to the scenario in quesiton:  

If you take a look in the getTheme.js buttonDisabled.container doesn't exist.

getTheme.js ~ line 151ish...
```
       buttonDisabled: StyleSheet.create(merge({
            text: {
                color: palette.disabledTextColor,
            },
        }, theme.buttonDisabled)),
```
In existing code base when buttonDisabled.container is called 'undefined' is returned.  

Honestly I'm unsure if adding it to support a (!raised && disabled) is necessary... Open to suggestion on this though.  May to override situations were primary=true and disabled=true? I'd have to test this situation but I think that would still work since I added the if(!disabled){ on line 67. 

Let me know wha toyu think ;)